packages/cssnano-preset-default/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "dist/index.js",
   "description": "Safe defaults for cssnano which require minimal configuration.",
   "scripts": {
-    "prepublish": "cross-env BABEL_ENV=publish babel src --out-dir dist --ignore /__tests__/"
+    "prepublish": "cross-env BABEL_ENV=publish babel src --out-dir dist --ignore /__tests__/ --copy-files"
```

Comment:
Why it is here?

Reply:
So that the custom order will get copied, babel normally ignores JSON files.